.github/workflows/build.yml

```diff
@@ -21,6 +21,55 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Cache Docker layers
+        id: cache-buildx
+        uses: actions/cache@v3
+        with:
+          key: buildx-${{ matrix.os }}-${{ hashFiles('jss.spec') }}
```

Comment:
Actually, this should be tomcatjss, right?

Reply:
Ah yes, fixed. Thanks!